### PR TITLE
espilitConvertedToVR

### PR DESCRIPTION
**Pull request by TrevorDev**

### CODE REVIEW

File: js/loader.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ var sceneChecked;
 var sceneLocation = "../../../Scenes/";
 
 // Babylon
-var engine = new BABYLON.Engine(canvas, true);
+var engine = new BABYLON.Engine(canvas, true, { preserveDrawingBuffer: true });
```
Comment (TrevorDev) [inline js/loader.js]: Are you aware of any downsides of me doing this for all scenes?

Comment (deltakosh) [reply js/loader.js]: No pb this is fine

File: Demos/GlowingEspilit/demo.js [hunk 2]
```diff
@@ -21,5 +20,9 @@
                 result.set(0, 0, 0, 0);
             }
         }
+        var light = new BABYLON.HemisphericLight("light1", new BABYLON.Vector3(0, 1, 0), scene);
```
Comment (deltakosh) [inline Demos/GlowingEspilit/demo.js]: Why the light?

**Comment by TrevorDev** [comment]
Minimally but it looks better IMO

**Pull request merged**